content/sensu-go/5.16/reference/etcdreplicators.md

```diff
@@ -336,6 +337,31 @@ spec:
 
 {{< /language-toggle >}}
 
+## Critical success factors for etcd replication
+
+Before you implement etcd replicators, review these must-know details that are critical to your success.
+
+**Bind your etcd listener to an external port that is *not* the default.**
+
+- Replication will not work if you bind your etcd listener to the default port.
+
+**Use only addresses that clients can route to for `etcd-client-advertise-urls`.**
+
+- If you use addresses that clients cannot route to for `etcd-client-advertise-urls`, replication my be inconsistent: it may work at first but then stop working later.
```

Comment:
```suggestion
- If you use addresses that clients cannot route to for `etcd-client-advertise-urls`, replication may be inconsistent: it may work at first but then stop working later.
```